src/app/games/lodestone/LodestoneRules.ts

```diff
@@ -21,15 +22,38 @@ export type LodestoneInfos = {
     moved: Coord[]
 }
 
-type LodestonePressurePlatePositionInformation = Record<LodestonePressurePlatePosition, [Coord, Coord, Direction]>;
+export interface PressurePlateViewPosition {
+    startForBigPlate: Coord,
+    startForSmallPlate: Coord,
+    direction: Direction,
+}
+
+export type PressurePlatePositionInformation =
+    MGPMap<LodestonePressurePlatePosition, PressurePlateViewPosition>;
 
 export class LodestoneRules extends Rules<LodestoneMove, LodestoneState, LodestoneInfos> {
-    public static readonly PRESSURE_PLATES_POSITIONS: LodestonePressurePlatePositionInformation = {
-        top: [new Coord(0, 0), new Coord(0, 1), Direction.RIGHT],
-        bottom: [new Coord(0, LodestoneState.SIZE-1), new Coord(0, LodestoneState.SIZE-2), Direction.RIGHT],
-        left: [new Coord(0, 0), new Coord(1, 0), Direction.DOWN],
-        right: [new Coord(LodestoneState.SIZE-1, 0), new Coord(LodestoneState.SIZE-2, 0), Direction.DOWN],
-    };
+    public static readonly PRESSURE_PLATES_POSITIONS: PressurePlatePositionInformation = MGPMap.from({
+        top: {
+            startForBigPlate: new Coord(0, 0),
```

Comment:
du coup ici ma question, comment (0, 0) est le commencement du la grosse prison sur un plateau de 8x8 quand la grosse fait 5 de large, c'est pas plutôt genre (1.5, -1) ????

Reply:
ça c'est la position utilisée par les rules pour enlever les pièces du plateau quand il y a un effondrement !

Reply:
threatenedCoordRange // the range of coord that this pressure place would take of if it would collapse
ou du genre ?